mrp_production_manual_close/models/mrp_production.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in root directory
+##############################################################################
+
+from openerp import models, fields, api
+
+
+class MrpProduction(models.Model):
+    _inherit = 'mrp.production'
+
+    production_done = fields.Boolean(string="Production Done")
+
+    @api.multi
+    def test_production_done2(self):
```

Comment:
OK, entendido. Pero lo de siempre, todo esto que explicáis por aquí se debe reflejar en el README de los módulos para que nosotros mismos primeros, y después los demás, sepamos de qué va el módulo.

@anajuaristi, todavía estoy esperando esa documentación funcional en los README...